pkg/client/client.go

```diff
@@ -164,24 +174,24 @@ func Initialize(
 				return
 			}
 			go monitorKeepClosedEvents(
-				ethereumChain,
-				keepAddress,
+				hostChain,
+				keep,
 				keepsRegistry,
 				subscriptionOnSignatureRequested,
 				eventDeduplicator,
 			)
 			go monitorKeepTerminatedEvent(
 				ctx,
-				ethereumChain,
+				hostChain,
 				networkProvider,
 				btcRecoveryAddress,
 				maxFeePerVByte,
 				clientConfig,
 				tssNode,
 				operatorPublicKey,
+				keep,
 				keepsRegistry,
 				eventDeduplicator,
-				keepAddress,
```

Comment:
Here's one of the main conflict sources: I was originally passing in a `keepAddress`, but master supplies a `keep`, so we don't need the address directly anymore